src/WebJobs.Script/Eventing/ScriptEventManager.cs

```diff
@@ -11,7 +11,13 @@ public sealed class ScriptEventManager : IScriptEventManager, IDisposable
         private readonly Subject<ScriptEvent> _subject = new Subject<ScriptEvent>();
         private bool _disposed = false;
 
-        public void Publish(ScriptEvent scriptEvent) => _subject.OnNext(scriptEvent);
+        public void Publish(ScriptEvent scriptEvent)
+        {
+            if (!_disposed)
```

Comment:
Agreed. I'll try reverting these and see if I can't track it back. Definitely we're getting lots of disposed exceptions all over the place though.